codegen/server/docs.go

```diff
@@ -0,0 +1,3 @@
+// Package server contains code generation algorithms to produce an example
+// server and client implementation for the transports defined in the design.
```

Comment:
does package server really produce an example client implementation?

Reply:
I named the package after the `Server` expression. It produces both the example server and the client. Do you want to change the package name to something else?

Reply:
We probably should yes. If the package generates the example code we could call it `example`?